README.md

```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.org/AntoineDao/ladybug.svg?branch=master)](https://travis-ci.org/AntoineDao/ladybug)
```

Comment:
@AntoineDao, can we put this under ladybug tools account for travis? e.g.: `https://travis-ci.org/*ladybug-tools*/ladybug`

Reply:
Yes absolutely. I left them under mine so you could see what the badges look like :)

Reply:
Yes absolutely. I left them under mine so you could see what the badges look like :)